cpp/src/compute/exec_backend.h

```diff
@@ -303,6 +313,7 @@ class GlutenResultIterator
   std::unique_ptr<GlutenIterator> iter_;
   std::shared_ptr<memory::GlutenColumnarBatch> next_;
   std::shared_ptr<ExecBackendBase> backend_;
+  int64_t exportNanos_;
```

Comment:
`collectMetrics` collect the metric from velox, all the origin metric comes from `VeloxPlanConverter` , but for `exportNanos_`, it needs to come with `GlutenVeloxColumnarBatch`, when revoke nativeFetchMetrics，it will be a part of Metrics